optaweb-tsp-ui/package.json

```diff
@@ -34,12 +34,19 @@
     "ajv": "^6.5.2",
     "enzyme": "^3.7.0",
     "enzyme-adapter-react-16": "^1.6.0",
-    "eslint": "^4.19.1",
-    "eslint-config-airbnb": "^15.1.0",
-    "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-jest": "^21.18.0",
-    "eslint-plugin-jsx-a11y": "^5.1.1",
-    "eslint-plugin-react": "^7.10.0",
-    "redux-devtools-extension": "^2.13.5"
-  }
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jest": "^21.27.1",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "react-test-renderer": "^16.6.0",
+    "redux-devtools-extension": "^2.13.5",
+    "redux-mock-store": "^1.5.3"
+  },
+  "browserslist": [
```

Comment:
@danielefiungo What is the effect of this configuration and how did you come up with the values? I think we need to document it so that following questions can be answered:
* What is the purpose of this piece of code?
* Can we remove it? / Under what conditions can it be removed?
* Can we change the values? / When do we need to update it?
A simple link to the related package documentation or an article explaining the usage might be helpful.